Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201305.java

```diff
@@ -359,8 +409,9 @@ public static String extractSSNIdentifier(PRPAMT201306UV02ParameterList params)
                             Identifier ssnId = new Identifier();
                             ssnId.setId(subjectId.getExtension());
                             ssnId.setOrganizationId(subjectId.getRoot());
-                            LOG.info("Created id from ssn identifier [organization=" + ssnId.getOrganizationId()
-                            + "][id=" + ssnId.getId() + "]");
+                            LOG.info("Created id from ssn identifier [organization=",
+                                ssnId.getOrganizationId()
+                                + "][id=" + ssnId.getId() + "]");
```

Comment:
use multiple log parameters for the logging
LOG.info("Created id from ssn identifier [organization: {} ][id: {} ]", 
ssnId.getOrganizationId(), ssnId.getId());